lib/active_admin/views/components/table_for.rb

```diff
@@ -11,7 +11,8 @@ def build(obj, *attrs)
         options = attrs.extract_options!
         @sortable = options.delete(:sortable)
         @collection = obj.respond_to?(:each) && !obj.is_a?(Hash) ? obj : [obj]
-        @resource_class = options.delete(:i18n)
+        @resource_class = options.delete(:resource_class)
+        @resource_class ||= options.delete(:i18n) # Deprecated
```

Comment:
Let's print a deprecation warning in this case, so that people can notice this and change their code.